src/main/java/com/icthh/xm/ms/entity/service/XmEntitySpecService.java

```diff
@@ -402,12 +336,12 @@ public TypeSpec findTypeByKey(String key) {
     @SneakyThrows
     private void processUniqueFields(Map<String, TypeSpec> types) {
         for (TypeSpec typeSpec: types.values()) {
-            if (isBlank(typeSpec.getDataSpec())) {
+            if (StringUtils.isBlank(typeSpec.getDataSpec())) {
```

Comment:
it's mistake? or why you rmove static import and decrease readability

Reply:
Explicit is better than implicit. Please think about [this ideas](https://www.python.org/dev/peps/pep-0020/)